students/ltrisolini/Lesson01/break_me.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+"""
+Lesson1, Task 1: Explore Errors
+Lorenzo Trisolini
+"""
+
+
+def NameError():
+    print(hello)
+
+def SintaxError():
+    print( 2 / 2 ))
+
+def typeerror():
+    a= 'Hello'
+    a=a + ' World'
+    a=a +5
+    print(a)
+    
+def attributeerror():
```

Comment:
some python style tips:
* keep an eye out for overwriting python built-ins like `NameError` you will notice different syntax highlighting
* function names (and var names) should follow snake_case convention -> `syntax_error`
* multiple words should be separated by underscores -> `type_error`